src/string-indices.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+export default function StringIndices(input, index) {
+  let toArray = input.toString().split(' ')
+
+  if(index >= toArray.length) {
```

Comment:
this should be `if(index > toArray,length)` If it is `>=` then if the user requests the last index it returns an empty string. (e.g. as it's written `StringIndicies('cash me outside how bow dah', 6) -> ''` but it should `-> dah`